tests/unit/Commands/SyncWoocommerceProductsTest.php

```diff
@@ -375,6 +388,18 @@ public function testConfigurableProductImport()
             );
 
             $this->assertProduct($original, $wc_configurable_product);
+
+            $originalSeoTitle = $original->get('flat_product.seo_title');
+            $seoTitle = $wc_configurable_product->get_meta('wpseo_title');
```

Comment:
```suggestion
            $seoTitle = $wc_configurable_product->get_meta('skseo_title');
```

We should use our own prefix not the one from other seo plugins.